app/models/post.rb

```diff
@@ -3,6 +3,9 @@ class Post < ApplicationRecord
   has_many :comments, dependent: :destroy
   belongs_to :author, class_name: 'User', foreign_key: :author_id
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comment_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] According to the requirements, ```LikesCounter```  must be an integer greater than or equal zero so kindly add a validation for it.